.changeset/fifty-buses-hope.md

```diff
@@ -0,0 +1,23 @@
+---
+"@comet/admin": major
+---
+
+Fix styling of `Alert` according to Comet DXP design
+
+Remove styling for `text` variant of Buttons used in `Alert`.
+Use `variant="outlined"` instead.
+
+```diff
+        <Alert
+    // ...
+    action={
+-       <Button variant="text" startIcon={<ArrowRight />}>
+-           Action Text
+-       </Button>
++       <Button variant="outlined" startIcon={<ArrowRight />}>
++           Action Text
++       </Button>
```

Comment:
Only the one line needs to be changed 🙂 

```suggestion
-       <Button variant="text" startIcon={<ArrowRight />}>
+       <Button variant="outlined" startIcon={<ArrowRight />}>
            Action Text
        </Button>
```

Reply:
48e32d4b4686d5626e6904102f219870eb50d178